### PR TITLE
Add The Ur-Quan Masters 0.7

### PR DESCRIPTION
**Pull request by bashu**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

The Ur-Quan Masters is a science fiction video game

### CODE REVIEW

File: Casks/uqm.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'uqm' do
```
Comment (vitorgalvao) [inline Casks/uqm.rb]: Token reference was not followed.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/uqm.rb (hunk 1 above)
Comment (bashu) [inline Casks/uqm.rb]: @vitorgalvao fixed

**Review by vitorgalvao — APPROVED** [review]

File: Casks/the-ur-quan-masters.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-cask 'ur-quan-masters' do
+cask 'the-ur-quan-masters' do
```
Comment (bashu) [inline Casks/the-ur-quan-masters.rb]: @vitorgalvao but why "The Battle for Wesnoth" has cask named "wesnoth"

Comment (vitorgalvao) [inline Casks/the-ur-quan-masters.rb]: You’ve made cask submissions before. Have you never read the token reference, despite ticking that box?

This token is dependent on the name of the app bundle.

**Pull request merged**